locale/zh_Hans/LC_MESSAGES/django.po

```diff
@@ -1647,15 +1630,17 @@ msgstr "专业型"
 
 #: meta/views.py:259 meta/views.py:336
 msgid "The highest average number of takedowns"
-msgstr ""
+msgstr "平均击杀数最高"
 
+# Is this highest mean lowest?
+# What is the difference between kills and takedowns?
```

Comment:
@jeklau If you open any match page, Statistics tab (for example, https://masterleague.net/match/2394/) you'll see that numbers of kills for any player is not too much. Kill is count only when the player made the last hit on enemy and after that hit enemy died. The kill applies only to one hero, while assists to all, who made the damage to enemy.

For example, while teamfight died one enemy player. In this teamfight involved 4 members of your team. 3 of them get assists and only 1 (who made the last hit) count a kill. Thats why number of kills is not too big, rather than number of assists or takedowns.